packages/template-typescript-minimal/app/ssr.js

```diff
@@ -8,6 +8,7 @@
 const path = require('path')
 const {getRuntime} = require('@salesforce/pwa-kit-runtime/ssr/server/express')
 const pkg = require('../package.json')
+const message = require('./typescript-file').default
```

Comment:
It's weird to import something and not use it.

```suggestion
// Example import of a TypeScript file
require('./typescript-file')
```
or
```suggestion
// Example import of a TypeScript file
const message = require('./typescript-file').default
console.log(message)
```
(and remove the `console.log` from the other file)

Reply:
Addressed